src/node_crypto.cc

```diff
@@ -155,7 +155,6 @@ std::string extra_root_certs_file;  // NOLINT(runtime/string)
 X509_STORE* root_cert_store;
 
 // Just to generate static methods
-template class SSLWrap<TLSWrap>;
```

Comment:
@danbev Actually, I’m a bit confused. :smile: For a linker error to be generated, the compiler would have first needed to be told that there are explicit instantiations generated elsewhere; but I don’t see anything like that?